tests/dryrun_test.go

```diff
@@ -149,34 +148,30 @@ var _ = Describe("[sig-compute]Dry-Run requests", decorators.SigCompute, func()
 	})
 
 	Context("VirtualMachines", func() {
-		var vm *v1.VirtualMachine
-		resource := "virtualmachines"
-
-		newVM := func() *v1.VirtualMachine {
-			vmiImage := cd.ContainerDiskFor(cd.ContainerDiskCirros)
-			vmi := tests.NewRandomVMIWithEphemeralDiskAndUserdata(vmiImage, "echo Hi\n")
-			vm := libvmi.NewVirtualMachine(vmi)
-			return vm
-		}
+		var (
+			vm        *v1.VirtualMachine
+			namespace string
```

Comment:
Why do we need this variable?

Reply:
@xpivarc - previously, we got the ns from the deprecated function. the new way does not do it, so we need to send thens parameter when creating the vm.

Reply:
This can be `const` as far as I see

Reply:
If we want to use `testsuite.GetTestNamespace` (as we do all over our e2e), then in go we can't assigned function result to a constant. Unless you want to do something else here.